BotTimer.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Threading.Tasks;
+using System.Timers;
+using Discord.WebSocket;
+
+namespace WaterBot
+{
+    static class BotTimer
+    {
+        static SocketTextChannel _channel;
+        static Timer _timer;
+        public static void Start(SocketTextChannel channel)
+        {
+            _channel = channel;
+
+            // Init timer of 3s intervals
+            _timer = new Timer(3000);
+            // Bind event when interval elapses
+            _timer.Elapsed += OnTimer;
+            _timer.Enabled = true;
+        }
+
+        public static void Stop()
+        {
+            _timer.Enabled = false;
+        }
+
+        public static void OnTimer(Object s, ElapsedEventArgs e)
+        {
+            _channel.SendMessageAsync("WATER TIME 💦💦💦💦💦💦💦💦💦💦💦");
+        }
+
```

Comment:
Interface doesn't have a reset function. You might need one?

Reply:
I believe the timer won't be reset throughout the lifetime of the bot, the plan is to use a static timer to keep track of the time and fire messages for users based on their subscribed time. 

Say user A registers at 10:15 and user B registers at 10:17. Instead of having different timers for each user, we plan to use the same timer and have a subscription table where we can pick up the time for each user, decide whether their interval has elapsed, and throw out the message accordingly.